### PR TITLE
[ES] Translation of 05-2.md and 05-3.md.

### PR DESCRIPTION
**Pull request by olivetom**
Mostly all know words and terms. Please review. Thanks. Mauricio

### CODE REVIEW

File: docs/es/week05/05-2.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+typora-root-url: 05-3
+---
+-->
+---
+lang-ref: ch.05-2
+title: Técnicas de Optimización II
+authors: Guido Petri, Haoyue Ping, Chinmay Singhal, Divya Juneja
+date: 29 Febrero 2020
+typora-root-url: 05-3
+---
```
Comment (Atcold) [inline docs/es/week05/05-2.md]: This won't work.
There's no language info, translator, translation date.
Please, check other headers.
This part is processed by the program that builds the website.

**Comment by Atcold** [comment]
The beginning of your file should look something like
```
---
lang: es
lang-ref: ch.01
title: Semana 1
translation-date: 24 Mar 2020
translator: LecJackS
---
```

**Comment by Atcold** [comment]
See #165.

**Comment by olivetom** [comment]
Hi @Atcold . Solved the header as in  #165. Although no _typora-root-url_ was included like in the original english version.

File: docs/es/week05/05-2.md [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
--->
 ---
 lang-ref: ch.05-2
 title: Técnicas de Optimización II
 authors: Guido Petri, Haoyue Ping, Chinmay Singhal, Divya Juneja
-date: 29 Febrero 2020
-typora-root-url: 05-3
+date: 29 Feb 2020
+lang: es
+translation-date: 27 Feb 2020
```
Comment (Atcold) [inline docs/es/week05/05-2.md:7]:
```suggestion
translation-date: 27 Mar 2020
```

**Comment by Atcold** [comment]
Yeah, `typora-root-url` is for an editor you can use. It's not for the website.

**Pull request merged**